tests/p2p_sync_throttle_test.py

```diff
@@ -180,6 +181,7 @@ def extractPrometheusMetric(connID: str, metric: str, text: str):
             if len(connPorts) < 2:
                 # wait for sending node to be connected
                 errorLimit -= 1
+                time.sleep(0.5)
```

Comment:
Is 0.5 second big enough?

Reply:
It is the value used on other failures. Since it tries multiple times, seems like more than enough.